phpBB/phpbb/storage/storage.php

```diff
@@ -94,6 +97,8 @@ public function get_contents($path)
 	 *
 	 * @param string	$path	file/directory to check
 	 *
+	 * @throws \phpbb\storage\exception\not_implemented	When the adapter doesnt implement the method
+	 *
 	 * @return bool	Returns true if all files/directories exist, false otherwise
```

Comment:
Returns true if the file/directory exist, false otherwise (the type hint is string)